packages/blade/src/utils/isReactNative/platform.all.ts

```diff
@@ -0,0 +1,32 @@
+// THIS .all file is special only for platform.ts file. Do not use anywhere else
+// This file only contains type level code and putting any runtime code will cause errors since our
```

Comment:
So if I add a noop function here `const noop = () => {}` will it throw error in build time or run time?

Reply:
Unless you import & use them.  

For example
Let's say you add `moduleSuffix: [".all", ".native"]` 

Now when you import `import noop from './example'` typescript will resolve the `.all` file with correct types but if your bundler isn't setup to handle the .all file extension this will fail.

Reply:
Basically it would fail the at the build step itself then?

Reply:
Yeah right.